src/main/java/tools/map/making/MapPropertyWrapper.java

```diff
@@ -40,14 +40,12 @@
  * @param <T>
  *        parameters can be: Boolean, String, Integer, Double, Color, File, Collection, Map
  */
-@SuppressWarnings("unchecked")
```

Comment:
Ditto on the unchecked cast warnings within Eclipse.

In this case, the annotation only really needs to be on the `getValue()` method.